scripts/base/init-bare.zeek

```diff
@@ -617,6 +617,19 @@ export {
 	const add_missing_remote_network_timestamp: bool = F &redef;
 }
 
+module ConnTuple;
```

Comment:
I'm going to rename `ConnTuple` to `ConnKey`. Mostly as right now the implementation is heavy on using the `ConnKey` terminology, but the Zeek module and also directory structure uses `ConnTuple`.

I'm preferring the "key" terminology and a bit of chatting with @J-Gras  and @bbannier, there's also no strong contender for "tuple". The main reason that this is about a "key" use in the session map.

This will also apply to the plugin component type and directory names.